src/main/java/frc/robot/subsystems/tower/ArmImpl.java

```diff
@@ -25,13 +28,39 @@
         POSITION_TICKS = UnmodifiableBidiMap.unmodifiableBidiMap(positionTicks);
     }
 
-    // TODO set up TalonSRX for Position PID:
-    // https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages/blob/master/Java/PositionClosedLoop/src/main/java/frc/robot/Robot.java
     private final WPI_TalonSRX controller;
     private Position targetPosition = Position.STARTING_CONFIG;
 
+    //Arm PID
+    private final int kPIDLoopIdx;
+    private final Gains kGains;
+    private final int kTimeoutMs;
+    private static Boolean kSensorPhase;
+    private static Boolean kMotorInvert;
```

Comment:
There's no reason for these to be `Boolean` (capital B, boxed type) rather than primitive `boolean`. And in fact they are never initialized, so they are currently null even.

Also, no apparent reason for them to be `static`.

Reply:
I'll change that, I had no idea there was even a difference in java